README.md

```diff
@@ -22,6 +22,10 @@ extension UIViewController {
 
     @AssociatedObject(.OBJC_ASSOCIATION_RETAIN_NONATOMIC)
     var text = "text"
+
+    static var customKey = ""
+    @AssociatedObject(.OBJC_ASSOCIATION_RETAIN_NONATOMIC, key: customKey)
```

Comment:
When `customKey` was defined as a let instead of a var, `&customKey` would cause an error, so I thought it would be better to receive the argument as an `UnsafeRawPointer`.

Reply:
I tried this before. Xcode give a error. Xcode think this is a inout symbol rather than pointer. I don't know why.

Reply:
It certainly did not work well.

As an alternative, I am wondering if we could stop generating the key property and use the following as the key

```swift
unsafeBitCast(Selector(#function), to: UnsafeRawPointer.self)
```

I tried it quickly and it appears to be working, but it needs to be more checked.

Reply:
It may work. But is the setter selector the same as the getter?

Reply:
Since it seems possible to obtain a unique address for a given string, I am also considering the following implementation.

```swift
let name = context.makeUniqueName(identifier.text).text
let associatedKey = """
unsafeBitCast(NSSelectorFromString((\"\(name)\")), to: UnsafeRawPointer.self)
"""
````